versions/3.0.4.md

```diff
@@ -121,7 +121,9 @@ In order to preserve the ability to round-trip between YAML and JSON formats, YA
 
 An OpenAPI Description (OAD) MAY be made up of a single document or be divided into multiple, connected parts at the discretion of the author. In the latter case, [Reference Object](#reference-object) and [Path Item Object](#path-item-object) `$ref` keywords, as well as the [Link Object](#link-object) `operationRef` keyword, are used.
```

Comment:
@mikekistler I like the first part of this, but I would prefer not to introduce "document identifier" given that "document identifier" is not a term I recall ever hearing.  The various reference keywords and the sections on URIs and URLs explain all of that just fine.  The reason that last sentence is there is because too many people aren't aware that the Path Item `$ref` is distinct, and the Link Object `operationRef` is similar.  Actually, the Discriminator Object's `mapping` can work with URIs as well although it's more complicated 🤔

Reply:
I didn't know the right terminology so I just took a stab. What I was trying to get across with the document identifier is that refs may be internal and these should not be considered to impose a "multi-document" OAD. Happy to rephrase to get this across.

Reply:
>  What I was trying to get across with the document identifier is that refs may be internal and these should not be considered to impose a "multi-document" OAD. Happy to rephrase to get this across.

I think this "multiple, connected documents" wording (with your added word "documents") gets this across sufficiently.  The listing of reference keywords is explicitly dependent on that ("In the latter case").

Reply:
This sentence was an error in the previous comment:

> The listing of reference keywords is explicitly dependent on that ("In the latter case").

What I mean is that the wording "multipe, connected documents" is _not_ dependent on the next sentence about reference objecs and keywords.

It might be good to just separate out that sentence into a new paragraph noting that any of the listed objects/keywords are able to connect additional documents.  Or something like that?